packages/arb-token-bridge-ui/tests/e2e/specs/txHistory.cy.ts

```diff
@@ -69,7 +69,8 @@ describe('Transaction History', () => {
       // search for invalid address substring
       cy.findByPlaceholderText(DEPOSIT_SEARCH_IDENTIFIER)
         .typeRecursively('0xx') // 0xx is invalid tx hash
-        .then(() => {
+        .then($input => {
```

Comment:
The tx history height got larger now and we need to scroll it to view. Otherwise pending txs cover most of the screen and the required text is not in view.